app/src/main/res/layout/character_fragment_dialog.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.globant.fragments.CharacterFragmentDialog">
+
+    <!-- TODO: Update blank fragment layout -->
```

Comment:
this TODO is still necessary?

Reply:
no, its not necessary jajajai removed it yet